lib/numorstring/port.go

```diff
@@ -50,13 +51,16 @@ func PortFromString(s string) (Port, error) {
 
 	parts := strings.Split(s, ":")
 	if len(parts) != 2 {
-		return Port{}, errors.New("invalid port format")
+		msg := fmt.Sprintf("invalid port format (%s)", s)
+		return Port{}, errors.New(msg)
```

Comment:
its not bugging me, I was just curious. Don't think there's a golang best practice for this, I was just wondering about the extra variable creation in memory